cfgov/jinja2/v1/ccdb-complaint/ccdb-search.html

```diff
@@ -18,7 +18,19 @@
 {%- endblock %}
 
 {% block content_main %}
-  <div id="ccdb-ui-root"></div>
+  <div id="ccdb-ui-root">
+    <div class="js-is-hidden wrapper u-mt60 u-mb60">
+      <div style="padding: 2em 1em;" class="m-notification m-notification__error m-notification__visible">
+        {{ svg_icon('error-round') }}
+        <div class="m-notification_content">
+          <div class="h4 m-notification_message">
+            This interactive tool is not supported in your web browser. Please use a newer browser or
+            <a style="text-decoration:underline" href="/data-research/consumer-complaints/#download-the-data">download the data</a>.
+          </div>
+        </div>
+      </div>
```

Comment:
Oh shouldn't we be using the existing notification template?https://github.com/cfpb/consumerfinance.gov/blob/03ce969697f3090e30d1d2a282e6980637a415f0/cfgov/jinja2/v1/_includes/molecules/notification.html

Usage example is at https://github.com/cfpb/consumerfinance.gov/blob/03ce969697f3090e30d1d2a282e6980637a415f0/cfgov/jinja2/v1/_includes/organisms/filterable-list.html#L43-L52

Reply:
The structure of that template is somewhat rigid and in this context prevented me from rendering the inline link. I didn't want to update that template to make all passed messages as `| safe` (because maybe some aren't).

Reply:
FWIW I recall something about that being by design, as we don't have notifications with inline links in the DS https://cfpb.github.io/design-system/components/notifications#information-base-notification-1 and the recommendation is to reword the notification to provide all links as a list at the end. I think it was about notification content coming from Wagtail and we didn't want a rich text block for the content.

But I also appreciate that this has already gone through the stakeholder and is very edge case.

Reply:
Or
<img width="552" alt="Screen Shot 2022-04-14 at 10 56 32 AM" src="https://user-images.githubusercontent.com/704760/163417143-e220b969-ffb9-4a7b-a1e0-fb7155503634.png">

Reply:
<img width="1332" alt="Screen Shot 2022-04-14 at 12 32 26 PM" src="https://user-images.githubusercontent.com/1558033/163433516-1340ddcc-1338-4f3d-84b0-c6b50fb1b3f3.png">

Reply:
(in 3834131)